.github/workflows/check-semver.yml

```diff
@@ -74,7 +72,7 @@ jobs:
 
       - name: install parity-publish
         # Set the target dir to cache the build.
-        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish -q
+        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish@0.8.0 -q
```

Comment:
0.7.0 is still used somewhere else i think.

Reply:
Yeah will bump the others now that i confirmed it works